src/Fantomas/Context.fs

```diff
@@ -883,6 +883,18 @@ let internal sepNlnConsideringTriviaContentBeforeWithAttributes (ownRange:range)
     |> Seq.exists (fun ({ ContentBefore = contentBefore }) -> hasPrintableContent contentBefore)
     |> fun hasContentBefore ->
         if hasContentBefore then ctx else sepNln ctx
+        
+let internal sepNlnConsideringTriviaContentBeforeAndAfter (moduleRange:range) ctx =    
+    match findTriviaMainNodeOrTokenOnStartFromRange ctx.Trivia moduleRange with
+    | Some node ->
+        if hasPrintableContent node.ContentBefore then
```

Comment:
Can be simplified to
```fsharp
        if hasPrintableContent node.ContentBefore || hasPrintableContent node.ContentAfter then
            ctx
        else
            sepNln ctx
```

Because I think this won't work if you have
```
module foo
// bar
// meh
```